testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDatabricksWithHttpMetastore.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.deltalake;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.tempto.BeforeMethodWithContext;
+import io.trino.tempto.assertions.QueryAssert;
+import io.trino.testng.services.Flaky;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.UUID;
+
+import static io.trino.tempto.assertions.QueryAssert.Row.row;
+import static io.trino.tests.product.TestGroups.DELTA_LAKE_DATABRICKS_HTTP_HMS;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_ISSUE;
+import static io.trino.tests.product.deltalake.util.DeltaLakeTestUtils.DATABRICKS_COMMUNICATION_FAILURE_MATCH;
+import static io.trino.tests.product.utils.QueryExecutors.onDelta;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestDatabricksWithHttpMetastore
+        extends BaseTestDeltaLakeS3Storage
+{
+    private String unityCatalogCatalogName;
+    private String unityCatalogExtLocation;
+
+    @BeforeMethodWithContext
+    @Override
+    public void setUp()
+    {
+        super.setUp();
+        unityCatalogCatalogName = requireNonNull(System.getenv("DATABRICKS_UNITY_CATALOG_NAME"), "Environment variable not set: DATABRICKS_UNITY_CATALOG_NAME");
+        unityCatalogExtLocation = requireNonNull(System.getenv("DATABRICKS_UNITY_CATALOG_EXT_LOC"), "Environment variable not set: DATABRICKS_UNITY_CATALOG_EXT_LOC");
+    }
+
+    @Test(groups = DELTA_LAKE_DATABRICKS_HTTP_HMS)
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testBasicTableReadWrite()
+    {
+        String schemaName = unityCatalogCatalogName + ".testBasicConnection";
+        String tableName = schemaName + ".testTable";
+        String trinoTableName = getTrinoTableName(tableName);
```

Comment:
Please use `randomNameSuffix`. Using static name leads to flaky tests when running on shared services.